woocommerce/woocommerce-android
# Update fluxc to SiteStore using coroutines

This PR updates fluxc to https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2018.

To test:
Smoke test the app including login - @planarvoid if you have tips on which parts we should test, please update the description

Update release notes:
